### DISCUSSION TITLE
Scratchblocks

### DISCUSSION THREAD

**Discussion by pikachub2005** [jeffalo/ocular#111] [Q&A] [answered]
How do you get the svgs? I want to use them for https://testing.scratchplus.repl.co/discuss/5/topic/543243.

**Comment by pikachub2005** [comment, marked as answer]
Thanks -_- guess I'll just look at https://github.com/jeffalo/ocular/blob/main/static/lib/scratchblocks.js

**Reply by jeffalo** [reply]
oops, sorry for missing this. i don't get email notifications on this repo.

scratchblocks.js is https://github.com/scratchblocks/scratchblocks
